test/library/packages/LinearAlgebra/performance/jacobi-perf.perfexecopts

```diff
@@ -0,0 +1,2 @@
+--n=512   # jacobi-n512
+--n=1024  # jacobi-n1024
```

Comment:
I'll be more comfortable if we remove this size as well. It came too close to the timeout.